examples/2_intermediate/mta.ipynb

```diff
@@ -0,0 +1,495 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "f4d848d1-dd58-4b75-9662-377cb66b5be9",
+   "metadata": {},
+   "source": [
+    "# Using CSP to analyze MTA data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6906efcb-324a-4bc1-b910-6d8905c0adb0",
+   "metadata": {},
+   "source": [
+    "The NYC Metropolitan Transportation Authority provides [an API for developers](https://api.mta.info), and we'll explore the MTA's realtime [GTFS-rt](https://developers.google.com/transit/gtfs-realtime) transportation data feed."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dacee7cb-d803-4eee-b135-c9ffd6ad2888",
+   "metadata": {},
+   "source": [
+    "In order to deal read and process the GTFS-rt data, we'll use the [nyct-gtfs](https://pypi.org/project/nyct-gtfs/) library, available from PyPI through\n",
+    "\n",
+    "```\n",
+    "pip install nyct-gtfs\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dcc5655d-3c45-4f96-842e-bb5f1b7d2127",
+   "metadata": {},
+   "source": [
+    "The MTA feed can be inspected as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "9409305a-a84a-493f-8eeb-9a7c1c2c81eb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from nyct_gtfs import NYCTFeed\n",
+    "\n",
+    "# Load the realtime feed from the MTA site\n",
+    "feed = NYCTFeed(\"https://api-endpoint.mta.info/Dataservice/mtagtfsfeeds/nyct%2Fgtfs\", api_key=\"\")\n",
+    "\n",
+    "# feed_id must be a valid feed URL or one of: \n",
```

Comment:
`feed_id` seems to not be used anywhere (also not a URL), maybe left over from an earlier version?
Also the current endpoint we're using is only for the 1-7 and S lines. It's fine to have it that way but let's just explain that in the text block the different lines have different endpoints, and we chose to use the largest feed (1-7 and S).